spec/rubocop/cop/rails/link_to_blank_spec.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Rails::LinkToBlank, :config do
+  ['link_to', 'link_to_if condition?, ', 'link_to_unless condition?, '].each do |link_helper_method|
```

Comment:
Examplary. All other specs in here could be wrapped like this (https://github.com/rubocop/rubocop-rails/pull/1289/commits/70db8e4f498ee3f3af4e23e5022717b4fa2d28e7)

Reply:
Can you divide this into three separate tests instead of using `each` loop?

Reply:
Sure, it will be three basically identically long blocks of tests then, but I understand if you prefer the explicitness.

Reply:
For example, redundant commas like in `with 'link_to_unless condition?, '` will be included in the text of `context`. To avoid this, it is acceptable to write three similar tests that only differ in the method.

Reply:
Okay, did that in the last commit for all the specs. If acceptable, I will squash the commits/drop the old one.